runtime/compiler/z/codegen/S390PrivateLinkage.hpp

```diff
@@ -39,9 +39,9 @@ namespace J9
 namespace Z
 {
 
-////////////////////////////////////////////////////////////////////////////////
+
 //  J9::Z::PrivateLinkage Definition for J9
-////////////////////////////////////////////////////////////////////////////////
+
```

Comment:
Lines that contain only slashes will be ignored by Doxygen, so there is no need to change these lines.  May I ask you to change these comments back to the following?

```
////////////////////////////////////////////////////////////////////////////////
//  J9::Z::PrivateLinkage Definition for J9
////////////////////////////////////////////////////////////////////////////////
```